### PR TITLE
fixing the sidebar scrollbar on firefox

### PR DESCRIPTION
**Pull request by martinraison**
@andrewconner @stkem Seems like sidebar width detection doesn't work in Firefox anymore. I hacked a solution that makes it look reasonable in both Chrome and Firefox. The downside is that the scrollbar is now always in the "thin" state (not sure why) but it actually still looks good, I think.

### CODE REVIEW

**Comment by andrewconner** [comment]
@2is10 Ideas?

**Comment by andrewconner** [comment]
@2is10 This is the symptom.

**Comment by martinraison** [comment]
hmm - looks like a missing overflow: hidden. I can't reproduce it on my Firefox
edit: I have Firefox 29

**Comment by 2is10** [comment]
​yes, missing overflow: hidden

**Comment by martinraison** [comment]
mmh - works fine for me in Firefox 30 too

File: kifi-backend/modules/shoebox/angular-black/src/tags/tags.styl [hunk 1]
```diff
@@ -216,6 +216,13 @@ input.kf-tag-rename
 .kf-scrollable-tags
   position: relative
 
+  // This is a hack to hide the scrollbar
+  // (cf. comments in scrollbar.js)
+  overflow: hidden
```
Comment (martinraison) [inline kifi-backend/modules/shoebox/angular-black/src/tags/tags.styl:221]: I added it here so not sure why it doesn't work for Andrew's browser

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular-black/src/tags/tags.styl:221]: This isn't merged yet. Do I have it?

Comment (martinraison) [inline kifi-backend/modules/shoebox/angular-black/src/tags/tags.styl:221]: oh I thought you had pulled the changes in some way (is there a quick way to "pull from a pull request"? - I mean without cloning my forked repo)

Comment (martinraison) [inline kifi-backend/modules/shoebox/angular-black/src/tags/tags.styl:221]: Merged - you can try if you want

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular-black/src/tags/tags.styl:221]: Yeah I can pull from yours (not sure if there's a quick way, but I have your remote as a local remote). But in this case, didn't.

**Pull request merged**